climakitae/explore/warming.py

```diff
@@ -100,7 +101,6 @@ def calculate(self):
         self.catalog_data = self.catalog_data.stack(
             all_sims=["simulation", "scenario"]
         ).squeeze()
-        self.catalog_data = self.catalog_data.dropna(dim="all_sims", how="all")
```

Comment:
This line is removed because in order to find all the simulations to drop that have NaNs in them, you have to load them all into memory, essentially >2x the total runtime of this method.